lamindb_setup/core/_hub_crud.py

```diff
@@ -89,15 +87,22 @@ def select_instance_by_id_with_storage(
     instance_id: str,
     client: Client,
 ):
-    response = (
+    # this won't find an instance without the default storage
+    data = (
         client.table("instance")
-        .select("*, storage!instance_storage_id_87963cc8_fk_storage_id(*)")
```

Comment:
Oh no, was this somehow forgotten before?

Reply:
Ok, this is only used during deletion; so it's not a big deal.

https://github.com/laminlabs/lamindb-setup/blob/db43bf308beb98c6834360c2950168fc1bb37c9d/lamindb_setup/core/_hub_core.py#L217

Maybe it was re-introduced at a later point or something similar.

No user will hit it.

Reply:
Yes, it was forgotten, but it is pretty specific thing, onlz called on deletion by id, so not really user facing.

Reply:
It's great you spotted it!